src/custom/pages/App/index.tsx

```diff
@@ -89,12 +89,23 @@ function createRedirectExternal(url: string) {
 
 const Loading = <LoadingWrapper>Loading...</LoadingWrapper>
 
+const getShowBannerState = (key: string) => {
```

Comment:
Should I move this to a Hook?

Reply:
Not big deal, because is a tiny function. However, it would ofcourse be better as a exported member of `SideBanner`. And it doesn't need to be hook, a pure function as u did is fine. 

In fact all the stuff relative to the banner could be there. This way you don't even need to expose `IS_SIDE_BANNER_VISIBLE_KEY` making this function simpler `getShowBannerState()` (no need for an argument).


Happy to apply the changes if you are overloaded, u already worked solo on this feature :)